sdk/tracing/azure-core-tracing-opencensus/README.md

```diff
@@ -0,0 +1,224 @@
+	<version>1.0.0-preview.4</version>
+</dependency>
+```
+
+### Configuring HTTP Clients
+When an HTTP client is included on the classpath, as shown above, it is not necessary to specify it in the client library [builders](#create-eventhubs-builders), unless you want to customize the HTTP client in some fashion. If this is desired, the `httpClient` builder method is often available to achieve just this, by allowing users to provide a custom (or customized) `com.azure.core.http.HttpClient` instances.
+
+For starters, by having the Netty or OkHTTP dependencies on your classpath, as shown above, you can create new instances of these `HttpClient` types using their builder APIs. For example, here is how you would create a Netty HttpClient instance:
+
+```java
+HttpClient client = new NettyAsyncHttpClientBuilder()
+	.port(8080)
+	.wiretap(true)
+	.build();
+```
+
+## Key concepts
+### Trace
+A trace is a tree of spans showing the path of work through a system. A trace on its own is distinguishable by a unique 16 byte sequence called a TraceID.
+### Span
+A span represents a single operation in a trace. A span could be representative of an HTTP request, a remote procedure call (RPC), a database query, or even the path that a code takes.
+
+## Examples
+The following sections provides examples of using the azure-core-tracing-opencensus plugin with some of the Azure Java SDK libraries:
+### Using the plugin package with Http Client libraries
+- Sync create a Secret using [azure-keyvault-secrets][azure-keyvault-secrets] with tracing enabled.
+    
+	Users can additionally pass the value of the current tracing span to the sdk's using key **"opencensus-span"** on the [Context][context] parameter of the calling method:
+	```java
+	import com.azure.identity.credential.DefaultAzureCredentialBuilder;
+	import com.azure.security.keyvault.secrets.SecretClientBuilder;
+	import com.azure.security.keyvault.secrets.SecretClient;
+	import com.azure.security.keyvault.secrets.models.Secret;
+	import io.opencensus.common.Scope;
+	import io.opencensus.trace.Tracer;
+	import io.opencensus.trace.Tracing;
+	import com.azure.core.util.Context;
+
+	import static com.azure.core.implementation.tracing.Tracer.OPENCENSUS_SPAN_KEY;
+
+	SecretClient client = new SecretClientBuilder()
+		.endpoint(<your-vault-url>)
+		.credential(new DefaultAzureCredentialBuilder().build())
+		.buildClient();
+
+	Tracer tracer = Tracing.getTracer();
+
+	try (Scope scope = tracer.spanBuilder("tracing-user-span").startScopedSpan()) {
+
+		// Create context with key "opencensus-span" and current tracing span as value
+		Context tracingContext = new Context(OPENCENSUS_SPAN_KEY, tracer.getCurrentSpan());
+
+		// Set secret and pass tracing context to pass the user-span to calling methods
+		Secret secret = secretClient.setSecretWithResponse(new Secret("secret_name", "secret_value", tracingContext));
+		System.out.printf("Secret is created with name %s and value %s \n", secret.getName(), secret.getValue());
```

Comment:
Instead of `\n`, it is preferred to use `%n` because it will give a platform dependent new line. so update to: `%s%n`